samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,13 +9,13 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def create_kserve_task(model_name, model_namespace, tfjob_op, model_volume_op):
</pre><p>My suggestion is to rename this to <code>create_serving_task</code> and have it be used for both kfserving and kserve since there isn't too much difference between this code block and the below code block. We want to minimize the amount of additional  code blocks we add.</p><p> </p><pre class="ql-syntax" spellcheck="false">def create_serving_task(model_name, model_namespace, tfjob_op, model_volume_op):


    api_version = 'serving.kserve.io/v1beta1'
    serving_component_url = 'https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kserve/component.yaml'


    # Uncomment the following two lines if you are using KFServing v0.6.x or v0.5.x.
    # api_version = 'serving.kubeflow.org/v1beta1"'
    # serving_component_url = 'https://raw.githubusercontent.com/kubeflow/pipelines/master/components/kubeflow/kfserving/component.yaml'


    inference_service = '''
apiVersion: "{}"
kind: "InferenceService"
metadata:
  name: {}
  namespace: {}
  annotations:
    "sidecar.istio.io/inject": "false"
spec:
  predictor:
    tensorflow:
      storageUri: "pvc://{}/"
'''.format(api_version, model_name, model_namespace, str(model_volume_op.outputs["name"]))


    serving_launcher_op = components.load_component_from_url(serving_component_url)
    serving_launcher_op(action="apply", inferenceservice_yaml=inference_service).after(tfjob_op)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/7355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>